assets/js/lib/settings.js

```diff
@@ -18,6 +18,11 @@ const DEFAULT_SETTINGS = {
   editor_font_size: EDITOR_FONT_SIZE.normal,
   editor_theme: EDITOR_THEME.default,
   editor_markdown_word_wrap: true,
+  custom_section: true,
```

Comment:
Since those are global settings, I would keep the names more explicit, like `custom_view_show_section`.